lib/players/human.rb

```diff
@@ -0,0 +1,18 @@
+module Players
+
+class Human < Player
+
+  def move(board)
+    print "#{@token} choose a position: "
+    input = gets.strip
+
+    if board.valid_move?(input)
```

Comment:
Ideally, this logic would be encapsulated within the board class, and Player::Human.move would only handle the gathering of input.